app/forms/green_lanes/moving_requirements_form.rb

```diff
@@ -1,12 +1,25 @@
 module GreenLanes
+  YEAR = 1
+  MONTH = 2
+  DAY = 3
+
   class MovingRequirementsForm
     include ActiveModel::Model
     include ActiveModel::Attributes
     include ActiveRecord::AttributeAssignment
 
     attribute :commodity_code, :string
     attribute :country_of_origin, :string
-    attribute :moving_date, :date
```

Comment:
I removed this because we needed a custom validation that triggers before the assignment.
I replaced it with two methods:
```
 def moving_date
and
def moving_date=
```